ignite/src/main/resources/emb-sql.xml

```diff
@@ -82,6 +85,12 @@
                     <entry key="field9" value="java.lang.String"/>
                   </map>
                 </property>
+                <property name="affinity">
```

Comment:
This should be a property of `CacheConfiguration` bean, not `queryEntities`

Reply:
fixed